### PR TITLE
fix(personal): Items created in personal workspace should not have expiry field present(SD-3877)

### PR DESCRIPTION
**Pull request by ioanpocol**
Also added the option on behave tests to specify that a field should not be present in response.

### CODE REVIEW

File: server/features/steps/steps.py [hunk 1]
```diff
@@ -88,6 +88,16 @@ def test_key_is_present(key, context, response):
         '"%s" should be empty or false, but it was "%s" in (%s)' % (key, response[key], response)
 
 
+def test_key_is_not_present(key, response):
```
Comment (akintolga) [inline server/features/steps/steps.py]: There's a similar function on this line...
https://github.com/superdesk/superdesk/blob/master/server/features/steps/steps.py#L887

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:  except comment for tolga

File: server/features/steps/steps.py (hunk 1 above)
Comment (ioanpocol) [inline server/features/steps/steps.py]: The solutions have a kind of overlapping but I think the new solution is more generally.

**Pull request merged**